src/main/resources/api/swagger-config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "sortParamsByRequiredFlag": false,
```

Comment:
huh. Is there a eway to specify these fields in the gradle file? Ok if not, but putting these with the rest of the swagger codegen configuration in build.gradle seems preferable.

Reply:
Look through all available options:
https://github.com/int128/gradle-swagger-generator-plugin#task-type-generateswaggercode seems configFIle is the only one works(configuration is something else).

And it only mentioned the json thing:
https://github.com/int128/gradle-swagger-generator-plugin#use-configuration-file